src/sql/workbench/contrib/query/browser/queryActions.ts

```diff
@@ -323,13 +323,53 @@ export class EstimatedQueryPlanAction extends QueryTaskbarAction {
 		}
 
 		if (this.isConnected(editor)) {
-			editor.input.runQuery(editor.getSelection(), {
-				displayEstimatedQueryPlan: true
-			});
+			let planOptions: ExecutionPlanOptions = editor?.input?.state?.isActualExecutionPlanMode ? { displayActualQueryPlan: true } : { displayEstimatedQueryPlan: true };
+			editor.input.runQuery(editor.getSelection(), planOptions);
 		}
 	}
 }
 
+/**
+ * Action class that toggles the actual execution plan mode for the editor
+ */
+export class ToggleActualExecutionPlanModeAction extends QueryTaskbarAction {
```

Comment:
I didn't give this action a shortcut since `CTRL`/`CMD` + `M`, can be used to quickly create an actual execution plan without needing to click on the toggle button. The new shortcut added is for the "Estimated Plan" button, and that one uses `CTRL`/`CMD` + `L` since that aligns with the estimated plans shortcut in SSMS.

Reply:
Sounds good, we can follow up with adding this later if we get feedback that customers want a keybinding.